diff --git a/dbus-listen.cabal b/dbus-listen.cabal
index 781e248..0aeed1c 100644
--- a/dbus-listen.cabal
+++ b/dbus-listen.cabal
@@ -26,7 +26,7 @@ executable dbus-listen
   main-is:             Main.hs
   -- other-modules:
   -- other-extensions:
-  build-depends:       base >=4.13 && <4.14
+  build-depends:       base
                      , dbus
                      , optparse-applicative
                      , process
